MewsSystems/mews-flutter
# feat!: [DX-1385] Add OptimusSpinner

#### Summary

- **[BREAKING CHANGE]** - Removed `OptimusCircleLoader`. Replaced with the `OptimusSpinner`
- Added `OptimusSpinner`
- Updated components that were using `OptimusCircleLoader` to use `OptimusSpinner`
- Updated Storybook

<details><summary>Preview</summary>



https://github.com/MewsSystems/mews-flutter/assets/9210422/5c71af94-be1e-4320-a99e-14b5b4b9981c


</details>


#### Testing steps

1. Open Spinner story
2. Check all sizes in the both themes.

#### Follow-up issues

None

#### Check during review

- Verify against Jira issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
